app/actor/PublicRoomActor.scala

```diff
@@ -0,0 +1,16 @@
+package actor
+
+import akka.actor._
+import scala.collection.mutable.ArrayBuffer
+
+object PublicRoomActor {
+  def props: Props = Props(new PublicRoomActor)
+}
+
+class PublicRoomActor extends WebSocketActor {
+  override var actorList: ArrayBuffer[ActorRef] = scala.collection.mutable.ArrayBuffer.empty
```

Comment:
Delete this line **IT WON'T COMPILE**
You don't need to override, it's already in the trait you extended

_**Please test your code before pushing your changes**_